crypto/fipsmodule/rand/rand.c

```diff
@@ -193,6 +273,7 @@ static void rand_get_seed(struct rand_thread_state *state,
     BORINGSSL_FIPS_abort();
   }
 
+  OPENSSL_STATIC_ASSERT(sizeof(entropy) % CRNGT_BLOCK_SIZE == 0, _);
```

Comment:
Just using `_` doesn't really give a useful error message. Shouldn't we be more descriptive?

Reply:
It's just replacing a `_` with something that actually makes sense?

Reply:
I think this is over.

This PR is just to merge latest BoringSSL code and conform the limit from `OPENSSL_STATIC_ASSERT`, which is from https://github.com/awslabs/aws-lc/pull/80.  

Asserts with empty error messages can make sense when the condition is short (self-explained). More verbose info can be added but then the case may not be "just replacing" or be kind of duplicate.

Besides, if each assert requires a  descriptive error msg, OPENSSL_STATIC_ASSERT may need to add the non empty validation on the `error`. But I don't think this is really necessary.